package.json

```
@@ -2,47 +2,106 @@
   "name": "little-window",
   "version": "1.0.0",
   "description": "A chatbot featured on Chayn websites, which quickly directs users to Chayn’s relevant internal or external resources",
-  "main": "config.js",
+  "author": "Chayn",
+  "license": "ISC",
```

(I'm guessing this was autogenerated / updated by the NestJS CLI) The top level license is MIT so we should change this.

```suggestion
  "license": "MIT",
```